packages/components/src/Enumeration/Header/HeaderInput.component.js

```diff
@@ -25,7 +25,7 @@ function getAction(action, index, getInternalInputRef) {
 		}
 	}
 
-	return (
+	return !action.disabled ? (
```

Comment:
you can add a test on disabled and if it is the case return null.
add if(action.disabled) { return null }

Reply:
test already done